oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
> The functions in question are useless outside of the context of using with with OAuth2WebServerFlow.

I don't think that's strictly true. As I mentioned above, a server-side application might want to run `code_challenge()` on the `code_verifier` sent by a client, in order to verify that they match.

At any rate, it's your call, so I already changed this stuff to all be private.